pkg/reconciler/shared/tektonconfig/trigger/trigger.go

```diff
@@ -170,3 +167,17 @@ func verifyNoTektonTriggerCR(ctx context.Context, clients op.TektonTriggerInterf
 	}
 	return nil
 }
+
+func GetTektonConfig() *v1alpha1.TektonConfig {
+	return &v1alpha1.TektonConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: v1alpha1.ConfigResourceName,
+		},
+		Spec: v1alpha1.TektonConfigSpec{
+			Profile: "all",
+			CommonSpec: v1alpha1.CommonSpec{
+				TargetNamespace: "tekton-pipelines",
+			},
+		},
+	}
+}
```

Comment:
if this is only used in testing, could you move this to the test file.

Reply:
sure will move it to test file

Reply:
will take it up later. could you create a test improvement issue. 
Im ok with merging this.

Reply:
yeah will create an issue for this 👍🏻